src/bzmesh.jl

```diff
@@ -65,15 +65,14 @@ function bzmesh_ir_wedge(kgrid_size, symops; kshift=[0, 0, 0])
     # Give the remaining symmetries to spglib to compute an irreducible k-Point mesh
     # TODO implement time-reversal symmetry and turn the flag to true
     is_shift = Int.(2 * kshift)
-    Stildes = [S' for (S, τ) in symops]
-    spg_rotations = permutedims(cat(Stildes..., dims=3), (3, 1, 2))
-    mapping, grid = pyimport("spglib").get_stabilized_reciprocal_mesh(
+    Stildes = [S for (S, τ) in symops]
```

Comment:
I'm confused by this: symops has rotations as S, and Stildes are transposes of S, so this line runs counter to the rest of the code. Is this done because C spglib is row major? In this case I think it'd be better if `spglib_get_stabilized_reciprocal_mesh` took column-major arrays and did the conversion internally (ie transpose twice)

Reply:
Or is it that spglib returns the S and not the Stilde?

Reply:
No spglib should return Stilde in our convention. It's the row-major versus column-major thing I think.

Reply:
I'm still confused on this actually, even though like this it seems everything works. According to the spglib documentation, transformations P, and coordinates x are defined such that 
P*x 
gives you the transformed coordinate. 

This together with my idea that C was column major like Julia, led me to believe that the rotations we got back from spglib should be in the order of DFTK, but somehow that turns out not to be true, hence the transpose in the get_symmetry part. 

Then originally I also had the transpose in the above line, but that then didn't work and produced garbage kpoints. This seems like an obvious inconsistency but I'm not clear on why that happens.

Reply:
Is this resolved @louisponet after the discussion yesterday evening?